### PR TITLE
Slice the reason in the audit log if it's too long to display

### PR DESCRIPTION
**Pull request by zBlakee**
don't mind the extra commits i don't know how to use git
despite the fact that it's literally 1 line i've probably stuffed it up

### CODE REVIEW

**Comment by ry0id** [comment]
absolutely sick

**Comment by 21Joakim** [comment]
Sick indeed

File: src/commands/ban.js [hunk 1]
```diff
@@ -14,6 +14,6 @@ module.exports = class BanCommand extends GenericModCommand {
       throw `Are you dumb? (run with \`sudo\` if you actually need to ban a bot)`;
     }
 
-    return guild.banMember(id, 0, reason);
+    return guild.banMember(id, 0, reason.length > 470 ? `${reason.slice(0, 470)}...` : reason);
   }
```
Comment (Aetheryx) [inline src/commands/ban.js]: You should probably slice them [here](https://github.com/DiscordBotList/bo/blob/master/src/structures/ModCommand.js#L52) instead so it works on all of the mod commands

**Comment by ry0id** [comment]
@Aetheryx he fixed :eyeshake:

**Comment by Aetheryx** [comment]
pinging in GitHub issues :wesmart:

**Pull request merged**